styles/styles.css

```diff
@@ -49,8 +54,39 @@
   --body-font-size-s: 14px;
   --body-font-size-xs: 12px;
   --body-font-size-xxs: 10px;
-  --line-height-l: 1.4em;
-  --line-height-m: 1.3em;
+
+  /* Line heights */
+  --line-height-2xs: 17px;
```

Comment:
Do we really use this many?

Reply:
Nope, I'm trying to eliminate some more -- might still get rid of one or two more before this is done.